src/components/SortTypeLink.tsx

```diff
@@ -0,0 +1,37 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  title: string
+};
+
+export const SortTypeLink: React.FC<Props> = ({ title }) => {
+  const [searchParams] = useSearchParams();
+  const sortByField = searchParams.get('sort');
+  const isReversed = searchParams.get('order') === 'desc';
+
+  const params = {
+    sort: (sortByField === title && isReversed) ? null : title,
```

Comment:
Move it to the variable & reuse it
```suggestion
    sort: ... ? null : title,
```